src/test/ui/lifetime-errors/ex1-return-one-existing-name-if-else-using-impl.nll.stderr

```diff
@@ -4,17 +4,16 @@ warning: not reporting region error due to nll
 LL |         if x > y { x } else { y } //~ ERROR lifetime mismatch
    |                    ^
 
-error[E0623]: lifetime mismatch
+error: unsatisfied lifetime constraints
   --> $DIR/ex1-return-one-existing-name-if-else-using-impl.rs:21:12
    |
 LL |     fn foo<'a>(x: &i32, y: &'a i32) -> &'a i32 {
-   |                   ----                 -------
-   |                   |
-   |                   this parameter and the return type are declared with different lifetimes...
+   |            --     - let's call the lifetime of this reference `'1`
+   |            |
+   |            lifetime `'a` defined here
 LL | 
 LL |         if x > y { x } else { y } //~ ERROR lifetime mismatch
-   |            ^^^^^ ...but data from `x` is returned here
+   |            ^^^^^ requires that `'1` must outlive `'a`
```

Comment:
interesting that we highlight the comparison here -- seems like a problem, though obviously not from this change...maybe we should file a bug? Presumably the *actual* span we should highlight is the use of `x`...

Reply:
This is mentioned on the `.nll.stderr`/`.stderr` comparison [here](https://github.com/rust-lang/rust/issues/52663#issuecomment-409032446).